package.json

```diff
@@ -39,16 +39,16 @@
     "extends": "enact-proxy/strict"
   },
   "dependencies": {
-    "@enact/core": "^4.1.2",
-    "@enact/i18n": "^4.1.2",
-    "@enact/spotlight": "^4.1.2",
-    "@enact/ui": "^4.1.2",
+    "@enact/core": "^4.1.4",
+    "@enact/i18n": "^4.1.4",
+    "@enact/spotlight": "^4.1.4",
+    "@enact/ui": "^4.1.4",
```

Comment:
These should be updated to ^4.5.0-alpha.1 once enact released.

Reply:
yes, i had this in mind :)